src/variables/classes/_hu-mix-blend-mode.scss

```diff
@@ -1,8 +1,8 @@
-$hu-blend-mode-alias: blend-mode !default;
+$hu-mix-blend-mode-alias: "" !default;
 
-$hu-blend-mode-modules: (base) !default;
+$hu-mix-blend-mode-modules: (base) !default;
```

Comment:
I wonder if we should change this to `()` to match `background-blend-mode`?  I can't imagine these are super regularly used